OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs

```diff
@@ -801,6 +801,22 @@ internal static void UpgradeWeaponRules(ModData modData, int engineVersion, ref
 					}
 				}
 
+				// Replaced projectile.TargetCenterPosition with WeaponInfo.TargetActorCenter
+				if (engineVersion < 20170622)
+				{
+					var instantHit = node.Value.Nodes.FirstOrDefault(n => n.Key == "Projectile" && n.Value.Value == "InstantHit");
+					if (instantHit != null)
+					{
+						var targetCenterPos = instantHit.Value.Nodes.FirstOrDefault(n => n.Key == "TargetCenterPosition");
+						if (targetCenterPos != null)
+						{
+							Console.WriteLine("The projectile TargetCenterPosition property was replaced with weapon TargetActorCenter property.");
+							instantHit.Value.Nodes.Remove(targetCenterPos);
+							node.Value.Nodes.Add(new MiniYamlNode("TargetActorCenter", "true"));
```

Comment:
Is this needed? This was recent bleed only change so no custom maps should use it. If they could use, other projectiles are ignored. Also it is assumed here, that it was always True, as true is hardcoded for TargetActorCenter.